Dockerfile

```diff
@@ -0,0 +1,35 @@
+
+
+
+FROM ubuntu:yakkety
+LABEL maintainer "peter@prioritylane.com"
+
```

Comment:
I've added a label to keep track of the image version – the Decrediton branch/tag you want to build is to be specified as a CLI argument though.